### DISCUSSION TITLE
Resourse usage

### DISCUSSION THREAD

**Discussion by pax0707** [Freika/dawarich#505] [Q&A] [answered]
Aloha.

Is this working as intended?
1st part is reindexing some 90k points so that would be understandable.
The the rest looks a bit excessive for something that received 1 set of coordinates every 5 minutes.
Posting all 4 containers, redis and sidekiq are the ones.

CPU
![image](https://github.com/user-attachments/assets/c9a1606e-9d8c-4d97-a701-abb6497f1b60)
![image](https://github.com/user-attachments/assets/37cff047-b1f1-48ae-858b-6c32c872764e)
![image](https://github.com/user-attachments/assets/22fdb132-10f3-440a-a61a-eeb2f9e155cd)
![image](https://github.com/user-attachments/assets/d9083870-67c5-4d52-bb13-632707012745)

RAM
![image](https://github.com/user-attachments/assets/2389327d-e911-4406-b471-3068f11bbdaa)
![image](https://github.com/user-attachments/assets/f46a59ad-84c8-47cb-bb1d-c80fa439e54b)
![image](https://github.com/user-attachments/assets/dd20534f-101b-485a-8891-e22ab3fa9207)
![image](https://github.com/user-attachments/assets/c3b3c802-0843-4d75-bb68-a17dca740e91)

**Comment by Freika** [comment, marked as answer]
I'm not sure why Redis spikes to 100% every now and then and uses that much memory, but otherwise everything looks pretty normal. Ruby is an interpreted language, which makes it of a memory-intensive kind.

**Comment by pax0707** [comment]
I checked with the current version and things look better.
/shrug

I checked current and older docker logs and nothing stood out.

![image](https://github.com/user-attachments/assets/edc70961-eab3-43ab-a461-caf9acae5a70)
![image](https://github.com/user-attachments/assets/27796888-efab-4bf9-94a7-9561283a3d87)

**Reply by extrafein** [reply]
Any idea, why  the sidekiq container is using constantly about 1,5% to 2% cpu power on an AMD EPYC 9554 64-Core Processor (4c/4t) while being idle (no geo-pushs, no http-data pulls)?

![image](https://github.com/user-attachments/assets/e0a3d993-cf7d-4966-8120-215745fbdba4)
